bazel/third_party/aes_siv/aes_siv.BUILD

```diff
@@ -17,4 +17,5 @@ cc_library(
     includes = ["."],
     visibility = ["//visibility:public"],
     deps = ["@openssl"],
+    copts = ["-O3", "-g"],
```

Comment:
Nit: put each element on its own line:
```python
copts = [
    "-O3",
    "-g",
]
```
More generally, [`buildifier`](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md) is a Bazel linter that will automatically "do the right thing" with respect to formatting Bazel BUILD files.

Reply:
This nit also applies to all the other third party dependency build files.

Reply:
done